doc/whats_new/v0.24.rst

```diff
@@ -158,9 +158,16 @@ Changelog
   :pr:`16289` by :user:`Masashi Kishimoto <kishimoto-banana>` and
   :user:`Olivier Grisel <ogrisel>`.
 
+:mod:`sklearn.manifold`
```

Comment:
please move this down so that it does not include the isotonic entry below

Reply:
Ah! Of course, my apologies.